src/Proto.Actor/Configuration/ActorSystemConfig.cs

```diff
@@ -50,5 +47,39 @@ public record ActorSystemConfig
         public ActorSystemConfig WithMetricsProviders(params IMetricsProvider[] providers) => this with {MetricsProviders = providers};
 
         public ActorSystemConfig WithDiagnosticsSerializer(Func<IActor, string> serializer) => this with {DiagnosticsSerializer = serializer};
+        
+        public ActorSystemConfig WithConfigureProps(Func<Props, Props> configureProps) => this with {ConfigureProps = configureProps};
+    }
+
+    //Not part of the contract, but still shipped out of the box
+    public static class ActorSystemConfigExtensions
+    {
+        public static ActorSystemConfig WithDeveloperReceiveLogging(this ActorSystemConfig self, TimeSpan receiveDeadline)
```

Comment:
Extension using the new ConfigureProps feature.
Allows developer to have console logging for receive duration. pinpointing slow actors